src/test/java/uk/gov/hmcts/reform/cwrdapi/controllers/StaffRefDataJobTitleControllerTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.hmcts.reform.cwrdapi.controllers.response.StaffRefJobTitleResponse;
+import uk.gov.hmcts.reform.cwrdapi.domain.RoleType;
+import uk.gov.hmcts.reform.cwrdapi.service.StaffRefDataService;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class StaffRefDataJobTitleControllerTest {
+
+    StaffRefDataService staffRefDataServiceMock;
+    StaffRefJobTitleResponse srJobTitleResponse;
+    ResponseEntity<Object> responseEntity;
+    @InjectMocks
+    private StaffRefDataController staffRefDataController;
+
+    List<RoleType> roleType = null;
+
+    @BeforeEach
+    void setUp() {
+        staffRefDataServiceMock = mock(StaffRefDataService.class);
+
+        srJobTitleResponse = StaffRefJobTitleResponse
+                .builder()
+                .jobTitles(Collections.emptyList())
```

Comment:
add list of couple of job titles instead of empty list

Reply:
@rajkatla-hmcts Modified